src/lib/hooks/useGetCurrentVoteList.tsx

```diff
@@ -5,42 +5,30 @@ import { VoteResultData } from '../../types/vote';
 import { picmeGetFetcher } from '../axios';
 
 const getKey = (cursorIdx: number, voteListData: AxiosResponse<VoteResultData>) => {
-  // console.log('getKey', cursorIdx, voteListData);
-
   if (cursorIdx === 0) return `vote/list/${cursorIdx}`;
   if (voteListData) return `vote/list/${voteListData.data.resCursorId}`;
-  return `vote/list/${cursorIdx}`;
+  return null;
 };
 const useGetCurrentVoteList = () => {
-  const { data, error, size, setSize } = useSWRInfinite<AxiosResponse<VoteResultData>>(getKey, picmeGetFetcher, {
-    errorRetryCount: 3,
-  });
-  // console.log('useGet', data);
-
-  const parseData = data?.map((item) => item.data);
-
-  // console.log('parseDaa', parseData);
-  // console.log(parseData?.map((item) => item.result));
+  const { data, isLoading, error, size, setSize } = useSWRInfinite<AxiosResponse<VoteResultData>>(
+    getKey,
+    picmeGetFetcher,
+    {
+      errorRetryCount: 3,
+    },
+  );
 
-  const parseResultList = parseData
-    ?.map((item) => item.result)
+  console.log(data);
+  const parseResultList = data
+    ?.map((item) => item.data.result)
```

Comment:
```suggestion
 const parseResultList = data
    ?.map(({ voteResultData }) => voteResultData.result)
```

이렇게 변수명도 이해하기 쉽게 바꿔주면 될것같아!